I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 35, in script
    show exmp3 with dissilve
  File "game/script.rpy", line 35, in <module>
    show exmp3 with dissilve
NameError: name 'dissilve' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 35, in script
    show exmp3 with dissilve
  File "C:\Users\Viktoria\Desktop\renpy-8.3.7-sdk\renpy\ast.py", line 1370, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\Users\Viktoria\Desktop\renpy-8.3.7-sdk\renpy\python.py", line 1218, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Viktoria\Desktop\renpy-8.3.7-sdk\renpy\python.py", line 1211, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 35, in <module>
    show exmp3 with dissilve
NameError: name 'dissilve' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.7.25031702
ArcompProject 1.0
Thu May  8 22:36:13 2025
